.github/workflows/optional-spec-validations.yml

```diff
@@ -41,7 +42,7 @@ jobs:
           npx -- @stoplight/spectral-cli@"${SPECTRAL_VERSION}" lint ./tmp/collection.json --ruleset=./validation/spectral.yaml
           popd
       - name: Create Issue
-        if: ${{ failure() }}
+        if: ${{ failure() && steps.spectral-validation.outcome == 'failure' }}
```

Comment:
can you explain a bit how this fix the issue? (newbie)

Reply:
sure

` failure() && steps.spectral-validation.outcome == 'failure'`
` failure()` check if the overall GH action has failed.
`steps.spectral-validation.outcome == 'failure'` makes sure that the step with the id `spectral-validation` has failed 

Only if both conditions are satisfied, the Create Issue step will run.

Before the fix, the only condition to create the issue was that the overall action has failed.